marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers/MintTokenEvent.hs

```diff
@@ -336,28 +344,35 @@ mkMintTokenIndexer dbPath = do
                INTO minting_policy_events (
                  slotNo,
                  blockHeaderHash,
-                 txIndexInBlock,
+                 blockNo,
                  txId,
+                 txIndexInBlock,
                  policyId,
                  assetName,
                  quantity,
                  redeemerHash,
                  redeemerData
               ) VALUES
-              (?, ?, ?, ?, ?, ?, ?, ?, ?)|]
+              (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)|]
       createMintPolicyEventTables =
-        [createMintPolicyEvent, createMintPolicyIdIndex, Sync.syncTableCreation]
+        [createMintPolicyEvent, createMintPolicyIdIndex]
       mintInsertPlans = [Core.SQLInsertPlan fromTimedMintEvents mintEventInsertQuery]
+      lastSyncPointQuery :: Core.GetLastSyncPointsQuery
+      lastSyncPointQuery = Core.GetLastSyncPointsQuery $ \limit ->
+        [sql|SELECT slotNo, blockHeaderHash
+             FROM minting_policy_events
+             ORDER BY slotNo DESC, blockNo DESC
+             LIMIT |]
+          <> fromString (show limit)
```

Comment:
It won't work because we don't have exactly 1 event in each block. We'll either have duplicate blocks or missing ones.
(It's why I added the atrocious sync table, and we'll have to redesign it in another way).